packages/cli/src/options/beaconNodeOptions/api.ts

```diff
@@ -89,4 +91,12 @@ export const options: CliCommandOptions<ApiArgs> = {
     type: "number",
     description: "Defines the maximum payload, in bytes, the server is allowed to accept",
   },
+
+  "rest.swaggerUI": {
+    type: "boolean",
+    description:
+      "Enable Swagger UI for API exploration at http://{address}:{port}/documentation. This should not be used in production.",
+    default: Boolean(defaultOptions.api.rest.swaggerUI),
+    group: "api",
+  },
```

Comment:
Potential other config options
- path
- server url (would be required if node runs behind proxy)

Depending on what use cases we see for this, might not be worth to add those now.